codecarbon/core/config.py

```diff
@@ -38,18 +39,25 @@ def parse_env_config() -> dict:
     }
 
 
-cwd = Path.cwd()
-global_config = configparser.ConfigParser()
-local_config = configparser.ConfigParser()
-env_config = configparser.ConfigParser()
-full_config = configparser.ConfigParser()
-
-global_config.read(Path("~/.codecarbon.config").expanduser().resolve())
-local_config.read(Path("./.codecarbon.config").expanduser().resolve())
-env_config.read_dict(parse_env_config())
-
-full_config.read_dict(global_config)
-full_config.read_dict(local_config)
-full_config.read_dict(env_config)
-
-config = full_config["codecarbon"]
+try:
```

Comment:
I added this in order for the user's process not to fail if they wrote faulty config files. But maybe we want it to fail? Thoughts?

Reply:
example faulty config:

```ini
measure_power_secs=10
```

^ is missing the section header `[codecarbon]` and `configparser` will fail

Reply:
caveat: the logger is defined in `emmissions_tracker.py` and so I only use prints here

Reply:
IMHO I'd rather let it crash.

Reply:
ok